src/main/scala/mesosphere/marathon/api/serialization/ContainerSerializer.scala

```diff
@@ -117,17 +123,31 @@ object ExternalVolumeInfoSerializer {
       .setName(info.name)
       .setProvider(info.provider)
 
-    info.size.foreach(builder.setSize)
+    info.size foreach builder.setSize
```

Comment:
We don't have a preference towards dotless syntax, so I suggest to revert this change and keep the changed lines small.